admin/worker/run_autoscaler.go

```diff
@@ -129,8 +129,21 @@ func (w *Worker) allRecommendations(ctx context.Context) ([]metrics.AutoscalerSl
 	return recs, true, nil
 }
 
+// shouldScale determines whether scaling operations should be initiated based on the comparison of
+// the current number of slots (originSlots) and the recommended number of slots (recommendSlots).
 func shouldScale(originSlots, recommendSlots int) bool {
+	// Temproray disable scale DOWN - Tony
+	if recommendSlots <= originSlots {
+		return false
+	}
+
 	lowerBound := float64(originSlots) * (1 - scaleThreshold)
 	upperBound := float64(originSlots) * (1 + scaleThreshold)
-	return float64(recommendSlots) < lowerBound || float64(recommendSlots) > upperBound
+	if float64(recommendSlots) >= lowerBound && float64(recommendSlots) <= upperBound {
+		return false
+	}
+
+	// TODO: Skip scaling for manually assigned slots
```

Comment:
@begelundmuller Not quite sure how to figure the slots that is specified by "manual intervene".

OR if is possible/necessary to have a new method for the DB `connect` struct to find the info from the Database?

Would you have some recommendations?

Reply:
I guess you would need to add a column with a flag in the database. Or use an annotation to indicate it (such as `has_sla` or something else – the annotation could be automatically added if a user calls `rill project edit --prod-slots ...`).